cmd/odo/help_test.go

```diff
@@ -29,8 +29,8 @@ Examples:
 
 	mainCommands = `Main Commands:
   build-images Build images
-  deploy       Deploy components
-  dev          Deploy component to development cluster
+  deploy       Run components in Deploy mode
```

Comment:
To be consistent with the help message for `dev`:

```suggestion
  deploy       Run your application on the cluster in the Deploy mode
```